### DISCUSSION TITLE
How can I make shared expenses accounted in totals properly?

### DISCUSSION THREAD

**Discussion by junoslukan** [mtotschnig/MyExpenses#1533] [Q&A]
Hi everyone,

I'm looking for some guidance on how to record shared expenses in MyExpenses. I'm referring to expenses that are incurred by multiple people, with one person paying for the entire amount and wanting to split it equally among the group. How could I input something like that into MyExpenses?

I've considered using both debts and split transactions, but I'm not sure if either option fully meets my needs.

In [this video](https://mastodon.social/@myexpenses/108160028317166243), split transactions are used to handle debts. This approach is close to what I'm looking for but:

- I don't think the debt feature is still available in the split transactions screen in the latest version; and 
- I don't know how such a transaction would be accounted for: would only "my part" (10$ in the video) be counted toward my total or the other parts as well?

Another suggestion I have come across was that I create a separate "account" for each person I'm splitting expenses with. This would allow me to keep track of how much is owed to and by each person, but I'm not sure how to ensure that these amounts are properly accounted for in my total expenses. For example, if I'm splitting expenses with one additional person, I would want half of the total for that person's account to be counted toward my own total expenses.

I'm interested in hearing how others approach these kinds of transactions in MyExpenses. Am I missing something obvious, or is there a recommended way to handle shared expenses that I'm not aware of? Any advice or insights would be greatly appreciated!

**Comment by mtotschnig** [comment]
> * I don't think the debt feature is still available in the split transactions screen in the latest version;

It should still work, and it does on my side:

![grafik](https://github.com/user-attachments/assets/22d6af1c-0c0d-4934-b3f2-530ad13ffada)

> * I don't know how such a transaction would be accounted for: would only "my part" (10$ in the video) be counted toward my total or the other parts as well?

My suggestion would be to either map the two other parts to a transfer category, or to configure unmapped transactions to be counted as transfers (in the Data chapter of the Settings).

> For example, if I'm splitting expenses with one additional person, I would want half of the total for that person's account to be counted toward my own total expenses.

Essentially representing debts via accounts should be roughly equivalent to using the debt feature. I do not understand why you would need to half the other person's balance, since it represents money that the person owes you, or that you owe the other person?

**Reply by junoslukan** [reply]
Hi @mtotschnig,

Thank you very much for your response. I see I missed the category split, you were of course correct that it is still available.

To clarify my use case: I want to track shared expenses, similar like what Splitwise or [Spliit](https://github.com/spliit-app/spliit) are doing. That is why I mentioned halving the expense (for the case of two people). Of course, I could do this for every transaction manually and then assign one part to my account and another to other(s), but I was wondering if there is a more elegant way of doing this. With more than two people, especially, it gets cumbersome to figure out who owes how much to whom.

**Reply by mtotschnig** [reply]
@junoslukan Thanks for the pointers, and sorry for the late reaction. I am interested in supporting the shared group expenses better, but I am not sure if My Expenses can fully replace specialized applications as the ones you mention, because My Expenses is focused around the personal expenses of you as the user, and I do not know how to represent the expenses of multiple members of a group inside one installation of the app.
What I could imagine on the one hand, is that we offer a shortcut for creating transactions that you want to split with others by setting up the debts, and creating split parts with automatically calculated amounts linked to the debts. 
On the other hand, if all members of the group use My Expenses and record their own expenses, and you would then want to settle their debts, there could be some online service which they could upload their individual reports to, and where the amounts that need to be transferred between them would be calculated.
